src/svelte/ShikiMagicMove.svelte

```diff
@@ -1,8 +1,8 @@
 <script lang="ts">
-	import { codeToKeyedTokens, createMagicMoveMachine } from '../core'
+	import { codeToKeyedTokens, createMagicMoveMachine } from '../core.mjs'
```

Comment:
This will break dev (the playground)

Reply:
I'm not sure how to fix it

Reply:
shipping `core.ts`, `renderer.ts` and `types.ts` will fix it. Because svelte files are compiled with the svelte version of the user of the library through vite. Do you think it will be feasible and possible through `unbuild`?

Reply:
for the moment i've added some other `fs.copyFile`, i don't know if there's a better way but this should make everything works both in the playground and in imported svelte projects.

Do you have a quick way to test this in react and vue too?

Reply:
I don't think shipping source files is a good idea. I feel it would be better if we could somehow compile the svelte file to pure JS, or rewrite the path on build

Reply:
if we could replace the extension at build time it would be great

Reply:
> I don't think shipping source files is a good idea. I feel it would be better if we could somehow compile the svelte file to pure JS, or rewrite the path on build

Compile svelte files to js is not the right approach because that would mean the compilation is stuck at the build moment. This basically means that if the user uses a different svelte version it might be uncompatible.

We can rewrite path on build although i see this solution as a bit more brittle than actually shipping the typescript files to allow them to be compiled at build time by the user.

The suggestion from @ghostdevv is also a good compromise imho.